kythe/typescript/indexer.ts

```diff
@@ -499,6 +494,20 @@ class StandardIndexerContext implements IndexerHost {
             parts.push('constructor');
           }
           break;
+        case ts.SyntaxKind.ImportClause:
+          // An import clause can have one of two forms:
+          //   import foo from './bar';
+          //   import {foo as far} from './bar';
+          // In the first case the clause has a name "foo". In this case add the
+          // name of the clause to the signature.
+          // In the second case the clause has no explicit name. This
+          // contributes nothing to the signature without risk of naming
```

Comment:
An import like `import {foo}` has a signature `foo`. No other entity can also have this signature because `foo` is now a global import declaration in the file and any local declarations of the same name will not be allowed by TypeScript, so `foo` will always be a unique signature.